README.md

```diff
@@ -49,6 +49,18 @@ export class SomeService {
   }
 }
 ```
+**Notes**: You  can **not** blindly pass any key/value as the 2nd argument(`msg`) of the `emit` function because there are two special reserved keys: `meta` and `options`. The `meta` is used by the default `OutboundEventSerializer` to attach extra informations; `options` is for setting the [options](https://github.com/Wiredcraft/nsq-strategies#new-producerconnectconfig-option) for the nsq message producing, see the sample below.
+
+```typescript
+
+    this.nsqProducer.emit(topic,
+      { foo: 1, bar: 2, meta: { component: 'XYZ'}, options: {retry: { retries: 3 } }
+    );
+    // The `emit` above will send a nsq message with payload as follow with 3 times retry strategy.
```

Comment:
Seems this comment should be moved up to line 57 at https://github.com/Wiredcraft/nestjs-nsq-transporter/pull/5/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R57

Reply:
No, the comment here is to explain the actual payload at line 60.

Reply:
But you didn't specify any options at line#60, so it sounds confusing to me.

Reply:
Oh, now I got your point, haha, different understanding. Then it should be fine.